docker/nginx.conf

```diff
@@ -117,7 +117,7 @@ http {
             proxy_set_header X-Real-IP $remote_addr;
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header X-Forwarded-Proto $scheme;
-            proxy_pass http://users:8180;
+            proxy_pass http://users.grpc-services:8180;
```

Comment:
do we want to change this URL?

Reply:
I need some advice on this change.
So far, in mainflux/k8s/nginx/nginx.conf there is a soft link to ../../docker/nginx.conf which means same configuration should be valid for both docker compose and Kubernetes deployments.
In the last commits introduced for Kubernetes  deployments (some weeks ago), specific namespaces were created. For the microservices using grpc it was decided to include them in grpc-services namespace.
Nginx however is belonging to a different namespace (default namespace). Domain names in Kubernetes include the name service together with the namespace. Hence if we have two micro-services belonging to different namespaces, both of them must use the complete domain name (which is not needed if they belong to the same namespace)
So, I guess there are two options:
1) All microservices are created in the same namespace
2) I remove the symbolic link and from now on there are two different versions of nginx.conf, one for Kubernetes deployment and another one for Docker Compose.
I would suggest the second option, but don't know if you have any other preference

Reply:
@juanmagal  2 is the option one. The other one will brake the docker composition. You can try, do `make run` from root.

Reply:
Ok, I have removed the link and created two different nginx.conf file: one for docker composition and another one for k8s.